app/models/manageiq/providers/azure/cloud_manager/provision.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::Azure::CloudManager::Provision < ManageIQ::Providers::CloudManager::Provision
+  include_concern 'Cloning'
+  include_concern 'StateMachine'
```

Comment:
The state_machine.rb file in this PR is empty.  You do not need that file or the include_concern unless you are overriding something.